core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -1095,139 +2124,215 @@ describe('Translation Suggestion Review Modal Component', function () {
-    function () {
-      const reviewable = true;
-      const subheading = 'topic_1 / story_1 / chapter_1';
+  describe('when viewing suggestion', function () {
+    const reviewable = false;
+    const subheading = 'topic_1 / story_1 / chapter_1';
 
-      const suggestion1 = {
-        suggestion_id: 'suggestion_1',
-        target_id: '1',
-        suggestion_type: 'translate_content',
-        change_cmd: {
-          content_id: 'hint_1',
-          content_html: ['Translation1', 'Translation2'],
-          translation_html: 'Tradução',
-          state_name: 'StateName',
-          cmd: 'edit_state_property',
-          data_format: 'html',
-          language_code: 'language_code',
-        },
-        exploration_content_html: ['Translation1', 'Translation2 CHANGED'],
-        status: 'rejected',
-        author_name: 'author_name',
+    const suggestion1 = {
+      suggestion_id: 'suggestion_1',
+      target_id: '1',
+      suggestion_type: 'translate_content',
+      change_cmd: {
+        content_id: 'hint_1',
+        content_html: ['Translation1', 'Translation2'],
+        translation_html: 'Tradução',
+        state_name: 'StateName',
+        cmd: 'edit_state_property',
+        data_format: 'html',
         language_code: 'language_code',
-        last_updated_msecs: 1559074000000,
-        target_type: 'target_type',
-      };
-      const suggestion2 = {
-        suggestion_id: 'suggestion_2',
-        target_id: '2',
-        suggestion_type: 'translate_content',
-        change_cmd: {
-          content_id: 'hint_1',
-          content_html: 'Translation',
-          translation_html: 'Tradução',
-          state_name: 'StateName',
-          cmd: 'edit_state_property',
-          data_format: 'html',
-          language_code: 'language_code',
-        },
-        exploration_content_html: 'Translation',
-        status: 'rejected',
-        author_name: 'author_name',
+      },
+      exploration_content_html: ['Translation1', 'Translation2 CHANGED'],
+      status: 'rejected',
+      author_name: 'author_name',
+      language_code: 'language_code',
+      last_updated_msecs: 1559074000000,
+      target_type: 'target_type',
+    };
+    const suggestion2 = {
+      suggestion_id: 'suggestion_2',
+      target_id: '2',
+      suggestion_type: 'translate_content',
+      change_cmd: {
+        content_id: 'hint_1',
+        content_html: 'Translation',
+        translation_html: 'Tradução',
+        state_name: 'StateName',
+        cmd: 'edit_state_property',
+        data_format: 'html',
         language_code: 'language_code',
-        last_updated_msecs: 1559074000000,
-        target_type: 'target_type',
-      };
+      },
+      exploration_content_html: 'Translation',
+      author_name: 'author_name',
+      language_code: 'language_code',
+      last_updated_msecs: 1559074000000,
+      status: 'status',
+      target_type: 'target_type',
+    };
+    const suggestion3Obsolete = {
```

Comment:
Can we have meaningful names? Ditto elsewhere.

Reply:
Done, named it to obsoleteSuggestion. [Commit: ](https://github.com/oppia/oppia/pull/19954/commits/4915cede39b71c0aab95ed9a6151c22fea08d9c5)